Dockerfile

```diff
@@ -1,24 +1,10 @@
-FROM alpine:3.4
-
-RUN apk add --no-cache curl python
-
-ENV GOOGLE_CLOUD_SDK_VERSION=161.0.0
-
-# Install the gcloud SDK
-RUN curl -fsSLo google-cloud-sdk.tar.gz https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-$GOOGLE_CLOUD_SDK_VERSION-linux-x86_64.tar.gz
-RUN tar -xzf google-cloud-sdk.tar.gz
-RUN rm google-cloud-sdk.tar.gz
-RUN ./google-cloud-sdk/install.sh --quiet
+FROM google/cloud-sdk:latest
```

Comment:
Can you set this to the alpine image?
Sorry, should have been more clear in my previous comment.

`latest` on its own is 600+mb.

https://hub.docker.com/r/google/cloud-sdk/tags/

Reply:
This has been pushed.